source/manual/access-to-licensing-for-third-parties.html.md

```diff
@@ -77,7 +77,7 @@ environments is as follows:
 
 Accessing machines using SSH in production and staging requires Production Admin Access. Third parties are usually only given Production Deploy Access, which only allows SSH in the integration environment.
 
-Follow the [instructions for connecting to a machine via SSH](/manual/howto-ssh-to-machines.html#connecting-with-plain-ssh). The machine classes you will need are `licensing_frontend` and `licensing_backend`. You will need to be on the VPN.
+The machine classes you will need are `licensing_frontend` and `licensing_backend`. You will need to be on the VPN.
```

Comment:
If we're removing the SSH doc, I'd be tempted to add an inline command here for the Licensify devs:

```suggestion
The machine classes you will need are `licensing_frontend` and `licensing_backend`. You will need to be on the VPN.

Connect to the instance by running, for example, `gds govuk connect -e production ssh licensing_backend`.
```